OpenRA.Mods.Common/Warheads/DestroyResourceWarhead.cs

```diff
@@ -23,7 +23,7 @@ public class DestroyResourceWarhead : Warhead
 		public readonly int[] Size = { 0, 0 };
 
 		[Desc("Amount of resources to be removed. If negative or zero, all resources within the area will be removed.")]
```

Comment:
`byte` is unsigned so can't be negative.

Reply:
updated comment